presto-raptor/src/test/java/com/facebook/presto/raptor/TestRaptorIntegrationSmokeTest.java

```diff
@@ -142,19 +142,19 @@ public void testCreateBucketedTable()
 
         assertQuery("SELECT * FROM orders_bucketed", "SELECT * FROM orders");
         assertQuery("SELECT count(*) FROM orders_bucketed", "SELECT count(*) FROM orders");
-        assertQuery("SELECT count(DISTINCT \"$shard_uuid\") FROM orders_bucketed", "SELECT 50");
```

Comment:
Let's count both. We want to make sure that a bucketed insert creates exactly one shard per bucket.